hr_employee_calendar_planning/models/resource_calendar.py

```diff
@@ -1,14 +1,30 @@
 # Copyright 2019 Tecnativa - Pedro M. Baeza
+# Copyright 2021 Tecnativa - Víctor Martínez
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import fields, models
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
 
 
 class ResourceCalendar(models.Model):
     _inherit = "resource.calendar"
 
     active = fields.Boolean(default=True)
 
+    @api.constrains("active")
+    def _check_active(self):
+        for item in self:
+            total_items = self.env["hr.employee.calendar"].search_count(
+                [("calendar_id", "=", item.id)]
+            )
+            if total_items > 0:
+                raise ValidationError(
+                    _(
+                        "%s is related to %s employee(s) in calendar planing,"
+                        " you should change it first." % (item.name, total_items)
+                    )
```

Comment:
Never include the placeholder elements inside the call to translate

```suggestion
                    _(
                        "%s is used in %s employee(s). You should change them first."
                    ) % (item.name, total_items)
```